src/main/java/sirius/pasta/noodle/compiler/ir/InstanceCheck.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.pasta.noodle.compiler.ir;
+
+import parsii.tokenizer.Position;
+import sirius.pasta.noodle.OpCode;
+import sirius.pasta.noodle.compiler.Assembler;
+import sirius.pasta.noodle.compiler.CompilationContext;
+
+import javax.annotation.Nullable;
+
+/**
+ * Represents an <tt>instanceof</tt> check.
+ */
+public class InstanceCheck extends Node {
```

Comment:
InstanceCheck -> InstanceOf check? The current name is okay but with the suggested one it would show up in class search when searching for `instanceof`.